mods/cnc/maps/gdi06/gdi06.lua

```diff
@@ -108,15 +108,18 @@ WorldLoaded = function()
 		KillCounterHuntThreshold = 20
 	end
 
-	destroyObjective = player.AddPrimaryObjective("Destroy the Nod ********.")
+	destroyObjective = player.AddObjective("Destroy the Nod ********.", "Primary", false)
+	dummyObjective = enemy.AddObjective("Dummy objective.", "Primary", true)
 
 	Trigger.OnKilled(Airfield, function()
 		player.MarkCompletedObjective(destroyObjective)
+		enemy.MarkFailedObjective(dummyObjective)
 	end)
 
 	Utils.Do(NodBase, function(structure)
 		Trigger.OnKilled(structure, function()
-			player.MarkCompletedObjective(destroyObjective)
+			player.MarkFailedObjective(destroyObjective)
```

Comment:
Is the use of `MarkFailedObjective` intentional here? I'd have expected `MarkCompletedObjective`.

Reply:
Yes, because you need to destroy the airfield in order to actually complete your objective. This is basically the fake objective you requested in the first PR.